src/formatstring/widget/formatstring.js

```diff
@@ -2,369 +2,386 @@
+            this.localizedFormat = {};
+
+            if (config.localizedFormats) {
+                config.localizedFormats.forEach(lang.hitch(this, function (format) {
+                    if (format.locale == dojo.locale) {
+                        this.localizedFormat = format;
+                    }
+                }));
+            }
+        },
+        postCreate: function () {
+            this._setupWidget();
+            this._setupEvents();
+            this.attributeList = this.notused;
+        },
+        update: function (obj, callback) {
+            if (obj) {
+                this._contextObj = obj;
+                this._resetSubscriptions();
+                this._loadData();
+                if (this.releaseOnUpdate === true) {
+                    if (this.release === true) this._releaseDisplayObjects();
                 }
-            }));
-        }
-    },
-    postCreate: function () {
-        this._setupWidget();
-        this._setupEvents();
-        this.attributeList = this.notused;
-    },
-    update: function (obj, callback) {	
-        if (obj) {
-          this._contextObj = obj;
-            this._resetSubscriptions();
-            this._loadData();
-            if(this.releaseOnUpdate === true){
-                if(this.release === true) this._releaseDisplayObjects();
-            }					
-        }			  
-        callback();
-    },
-    _releaseDisplayObjects: function (){
-        if(this._releaseList.length > 0){
-            mx.data.release(this._releaseList);
-//			console.log(this.id+"released");
-        }
-    },
-    _setupWidget: function () {
-        this._wgtNode = this.domNode;
-        domClass.add(this._wgtNode, 'formatstring_widget');
-    },
-    _setupEvents: function () {
-        if (this.onclickmf) {
-            this.connect(this._wgtNode, "onclick", this.execmf);
-        }
-    },
-    _loadData: function () {
-        this.replaceattributes = [];
-        var referenceAttributeList = [],
-            numberlist = [],
-            i = null,
-            value = null;
-        for (i = 0; i < this.attributeList.length; i++) {
-            if (this._contextObj.get(this.attributeList[i].attrs) !== null) {
-                value = this._fetchAttr(this._contextObj, this.attributeList[i].attrs, this.attributeList[i].renderHTML, i,
-                    this.attributeList[i].emptyReplacement, this.attributeList[i].decimalPrecision, this.attributeList[i].groupDigits);
-                this.replaceattributes.push({
-                    id: i,
-                    variable: this.attributeList[i].variablename,
-                    value: value
-                });
-            } else {
-                //we'll jump through some hoops with this.
-                referenceAttributeList.push(this.attributeList[i]);
-                numberlist.push(i);
             }
-        }
+            callback();
+        },
+        _releaseDisplayObjects: function () {

```

Comment:
Whole function could be commented out as it not functional at Mendix 7
https://apidocs.mendix.com/7/client/mx.data.html#.release

Reply:
Done